projects/packages/password-checker/src/class-password-checker.php

```diff
@@ -348,29 +348,39 @@ protected function get_common_passwords() {
 	 * @return array user data.
 	 */
 	protected function get_other_user_data() {
-
 		if ( ! isset( $this->user ) ) {
```

Comment:
Should we be explicit in checking user_id vs user as that is what gets used in the get_userdata call?

Reply:
Fixed.